cogs/rss/converter.py

```diff
@@ -0,0 +1,38 @@
+"""
+Converts v2 RSS json file to v3 format
+"""
+import json
+
+UID = "5842647" # When typed on a T9 keyboard: luicogs
+BASE = \
+{UID: {"GLOBAL": {}, "GUILD": {}}}
+
+
+#In bot constants (this was changed while updating the v2 RSS)
+INTERVAL = "check_interval"
+CHANNEL_ID = "post_channel"
+RSS_FEED_URLS = "rss_feed_urls"
+
+
+V3JSON = BASE
+
+GUILD_ID = input("Please input your server ID: ")
+
+# Convert filter into v3 format.
+with open("settings.json") as v2Filters:
```

Comment:
I know this was a copy from my previous converter, but let's rename the variables so that they make sense in this context of RSS.